### DISCUSSION TITLE
Set Favorite Parameters In The Inspector

### DISCUSSION THREAD

**Discussion by Ledracarnos** [godotengine/godot-proposals#9213] [Editor]
To be honest, I haven't found a similar topic, but there probably is.
I've been thinking a lot about this issue.

It is sometimes difficult for the user to access the parameters, especially if there is a large hierarchy. 
In addition, it's easier to use a restricted parameter menu for non-developers (level artists). 

![image](https://github.com/godotengine/godot-proposals/assets/79541759/758b48eb-aacc-4416-90e9-e394b0cde259)
The idea is to add a way of defining favorite parameters.

Seeing FR3NKD's twitt, I looked for a suitable solution to this problem. 
![image](https://github.com/godotengine/godot-proposals/assets/79541759/39e860f8-88e7-46d4-8b65-28e653eeb110)

Basically, the user defines a list of strings, and godot compares this list with the names of the parameters.
The order of the parameters depends on the order of the list.

The advantage of doing this with a string list is that it can easily be shared as a resource. 
In addition, it's possible to imagine special characters for special formatting. 

The crucial question is where to put this list?

I don't really have an answer, but I hope to get your feedback on this idea.

**Comment by Calinou** [comment]
Thanks for the proposal! Consolidating in https://github.com/godotengine/godot-proposals/issues/4274.

**Reply by Ledracarnos** [reply]
Thank you!
I'm not familiar with github, Should I add this in the dedicated post directly?

**Reply by Calinou** [reply]
>  I'm not familiar with github, Should I add this in the dedicated post directly?

The proposal seems very similar, so I don't think you need to post what you said here in a comment on the other proposal.
